examples/addsvc/client/main.go

```diff
@@ -31,6 +36,10 @@ func main() {
 		thriftProtocol   = flag.String("thrift.protocol", "binary", "binary, compact, json, simplejson")
 		thriftBufferSize = flag.Int("thrift.buffer.size", 0, "0 for unbuffered")
 		thriftFramed     = flag.Bool("thrift.framed", false, "true to enable framing")
+
+		// Two OpenTracing backends (to demonstrate how they can be interchanged):
+		appdashHostport      = flag.String("appdash_hostport", "", "Enable Appdash tracing via an Appdash server host:port")
+		lightstepAccessToken = flag.String("lightstep_access_token", "", "Enable LightStep tracing via a LightStep access token")
```

Comment:
Got it. For the final version, please use `"appdash.addr"` and "lightstep.access.token"`.